package.json

```diff
@@ -49,7 +51,8 @@
     "tslint": "^6.1.3",
     "tslint-config-prettier": "^1.18.0",
     "tslint-plugin-prettier": "^2.3.0",
-    "typescript": "^4.3.2"
+    "typescript": "^4.3.2",
+    "zksync-web3": "^0.14.3"
```

Comment:
Should we be using `zksync-ethers`?

Reply:
can't as it uses ethersv6 and aave-v3-deploy uses v5.